lib/searchgov/style/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module SearchgovStyle
+  VERSION = '0.1.0'
```

Comment:
I would call this version 1.0.0. Not that it matters hugely, in this case, but.

Reply:
I had the same thought, but stuck with `'0.1.0'` for a couple of reasons:
-  `'0.1.0'` is the default version for new gems
- `0.x.x` versions are expected to have "rapid development and fast iteration", and do not promise backwards-compatibility

I expect to make several rounds of changes in the near future as I adapt this based for each subsequent repo. Once that's done, and we expect it to be more stable going forward, that seems like a good time to bump it to 1.0.0.